### DISCUSSION TITLE
github enterprise license

### DISCUSSION THREAD

**Discussion by almond-mixnut** [community/community#58430] [Enterprise]
### Select Topic Area

Question

### Body

hello, I have an organization planed with github enterprise.

but I can't reach enterprise license menu.

when I visit organization profile setting page, there are no license menu at all. 

I need a license file to setting up github enterprise server for our company's new project

plz help me to get enterprise license

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @almond-mixnut! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
